keyboards/ls_60/readme.md

```diff
@@ -0,0 +1,17 @@
+# LS_60
+
+![LS_60](https://www.hualigs.cn/image/6108151699aea.jpg)
```

Comment:
Please rehost this on Imgur. The link is dead.

Reply:
Please check the link, I found it available here.

Reply:
I got a PHP error before (no idea what it said though).
It's a good idea to host it on a more reliable service like imgur.

Reply:
I'm very sorry for the mistake just made. I will pull request to update the pictures in time after hosting them to imgur.Because Chinese mainland support for imgur is very bad, so this may take a little time.

Reply:
```suggestion
![LS_60](https://i.imgur.com/Cuy5W1H.png)
```
Okay, I've uploaded it for you to save some time :)

Reply:
Thank you very much! Is there anything else I need to change?